News article for after SR2020 Finals (#226)
srobo/website: _posts/2020-07-25-post-competition.md

title: JAMDynamics steal victory in Student Robotics 2020
---

Congratulations to JAMDynamics who take victory in a tie-breaker match after a
strong performance in the league and knockout stages.

The Challenge: *Two Colours*
----------------------------

{% include figure.html
           class="right"
           src="/images/content/news/sr2020/arena.png"
           caption="The virtual arena with its central raised platform" %}

Our game this year, _Two Colours_, challenged teams retrieve gold or silver
coloured tokens from on and around a raised platform in the centre of the arena.
Having these tokens within their scoring zone (i.e. their corner of the arena)
at the end of the match would earn them 3 points apiece. However, if a robot
gathered more than one token colour within their scoring zone, each token's
value drops from 3 points to 1 point. Full details of the game and the awards
available this year are available in the [rulebook][rulebook].

Our competition event this year was rather different to normal as COVID-19
prevented our usual weekend event. Instead the competition took place over
three weekends, with matches being live-streamed from within a simulator.

> While this changed many aspects of the competition the core challenge to create

Author: 6c4a4c3

posted online and at the Tech Days. Even during the virtual competiton, it was
great to see the robot code improving week to week. We hope you enjoyed the
competition as much as we did!

If you didn't take part, or you want to enter again next year, the
[sign up page]({{ site.baseurl }}/compete/) for next year's competition will be
up later in the year. Get a team together and start talking to your teachers now!

Results
-------

|        Prize          |            Team
|-----------------------|-----------------------------------------------
| 1st Place             | JAM: JAMDynamics
| 2nd Place             | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/)
| 3rd Place             | [LSS: Lawrence Sheriff School](http://www.lawrencesheriffschool.net/)
| <span class="has-explanation" title="The team that displays the most extraordinary ingenuity or elegant simplicity in the design of their robot">Committee Award</span> | [SWI: South Wilts Grammar School](https://www.swgs.wilts.sch.uk/)
| <span class="has-explanation" title="The rookie team who achieve the highest place in the league">Rookie Award</span> | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/)

> | <span class="has-explanation" title="The first rookie team1that demonstrates a moving robot to the community will be awardedwith an edible prize at the final competition">First Robot Movement</span> | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/) ([watch their robot](https://www.youtube.com/watch?v=KuFwqVyn_YA))

Reviewer: ```suggestion
| <span class="has-explanation" title="The first rookie team that demonstrates a moving robot to the community">First Robot Movement</span> | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/) ([watch their robot](https://www.youtube.com/watch?v=KuFwqVyn_YA))
```

> | <span class="has-explanation" title="The team that is judged to have the best online presence using the hashtag #srobo19">Online Presence</span> |  [MAI: Gymnasium Markt Indersdorf](https://mai-robotics.de/)

Reviewer: ```suggestion
| <span class="has-explanation" title="The team that is judged to have the best online presence using the hashtag #srobo2020">Online Presence</span> |  [MAI: Gymnasium Markt Indersdorf](https://mai-robotics.de/)
```

With a dominating 111 league points, Kenilworth Sixth Form not only placed
highest among the rookie teams but placed highest overall in the league.
Especially given the changes to the competition through the year, beating other

> more eperience teams is a remarkable feat.

Reviewer 2: ```suggestion
more experience teams is a remarkable feat.
```

> more experience teams is a remarkable feat.

Reviewer 2: Apologies my typo fix still was a typo
```suggestion
more experienced teams is a remarkable feat.
```

Author: e78f0cf

> more experience teams is a remarkable feat.

Reviewer: ```suggestion
more experienced teams is a remarkable feat.
```

           src="/images/content/news/sr2020/arena.png"
           caption="The virtual arena with its central raised platform" %}

Our game this year, _Two Colours_, challenged teams retrieve gold or silver
coloured tokens from on and around a raised platform in the centre of the arena.
Having these tokens within their scoring zone (i.e. their corner of the arena)
at the end of the match would earn them 3 points apiece. However, if a robot
gathered more than one token colour within their scoring zone, each token's
value drops from 3 points to 1 point. Full details of the game and the awards
available this year are available in the [rulebook][rulebook].

Our competition event this year was rather different to normal as COVID-19
prevented our usual weekend event. Instead the competition took place over
three weekends, with matches being live-streamed from within a simulator.
While this changed many aspects of the competition the core challenge to create
an autonomous robot remained the same.

The Final
---------

This year's final featured four teams who had all proven their capabilities
throughout the competition. All robots got off to a strong start, though

> JAMDynamics and Lawrence Sheriff School soon took a lead. However later in the

Reviewer 3: ```suggestion
JAMDynamics and Lawrence Sheriff School soon took a lead. However, later in the
```

Author: 0446bf7

match, the "poisoning" effect of having mixed tokens in a scoring zone changed
everything leaving all four teams with the same number of points.

<!-- Video of Final -->
<iframe
  class="center video"

>   src="https://www.youtube-nocookie.com/embed/XXX-TODO"

Reviewer 3: May need to wait up to a day before the stream finishes processing and can be downloaded for cutting

  frameborder="0"
  width="100%"
  allowfullscreen
  >
</iframe>

This tie for first place lead to a tie-breaker match, played between the same
four teams though with their starting corners changed. Once again the
reliability of these four robots showed, with each collecting a token inside the

> one minute mark. In the end it was JAMDynamics robot which was able to collect

Reviewer 3: ```suggestion
one minute mark. In the end it was JAMDynamics' robot which was able to collect
```
Not sure whether it's this or `JAMDynamics's`

Author: db7041e

three silver tokens, including one stolen from Hills Road Sixth Form College

> second teams' scoring zone, which took victory.

Reviewer 3: ```suggestion
second team's scoring zone, which took victory.
```

Author: d75917f

The Final
---------

This year's final featured four teams who had all proven their capabilities
throughout the competition. All robots got off to a strong start, though
JAMDynamics and Lawrence Sheriff School soon took a lead. However, later in the
match, the "poisoning" effect of having mixed tokens in a scoring zone changed
everything leaving all four teams with the same number of points.

<!-- Video of Final -->
<iframe
  class="center video"
  src="https://www.youtube-nocookie.com/embed/XXX-TODO"
  frameborder="0"
  width="100%"
  allowfullscreen
  >
</iframe>

This tie for first place lead to a tie-breaker match, played between the same
four teams though with their starting corners changed. Once again the
reliability of these four robots showed, with each collecting a token inside the
one minute mark. In the end it was JAMDynamics' robot which was able to collect
three silver tokens, including one stolen from Hills Road Sixth Form College
second team's scoring zone, which took victory.

<!-- Video of Tie breaker -->
<iframe
  class="center video"
  src="https://www.youtube-nocookie.com/embed/XXX-TODO"
  frameborder="0"
  width="100%"
  allowfullscreen
  >
</iframe>

> While we weren't able to see them in person, we saw some brilliant robot designs

Reviewer: They looked the same. I'd be tempted to either drop the "design" or mention that it was more about strategy design etc

```suggestion
While we weren't able to see them in person, we saw some brilliant robots
```

Author: Ah, I was trying to acknowledge the physical robots here.

Reviewer: ah, yeah definitely worth drawing focus to that, but worth being slightly more explicit

Author: 98bf644

| 2nd Place             | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/)
| 3rd Place             | [LSS: Lawrence Sheriff School](http://www.lawrencesheriffschool.net/)
| <span class="has-explanation" title="The team that displays the most extraordinary ingenuity or elegant simplicity in the design of their robot">Committee Award</span> | [SWI: South Wilts Grammar School](https://www.swgs.wilts.sch.uk/)
| <span class="has-explanation" title="The rookie team who achieve the highest place in the league">Rookie Award</span> | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/)
| <span class="has-explanation" title="The first rookie team1that demonstrates a moving robot to the community will be awardedwith an edible prize at the final competition">First Robot Movement</span> | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/) ([watch their robot](https://www.youtube.com/watch?v=KuFwqVyn_YA))
| <span class="has-explanation" title="The team that is judged to have the best online presence using the hashtag #srobo19">Online Presence</span> |  [MAI: Gymnasium Markt Indersdorf](https://mai-robotics.de/)

With a dominating 111 league points, Kenilworth Sixth Form not only placed

> highest among the rookie teams in the league but placed highest overall.

Reviewer: This reads almost like they took 1st.

```suggestion
highest among the rookie teams but placed highest overall in the league.
```

the design of their solution. Team SWI from South Wilts Grammar School earned
this prize for an impressive path-finding algorithm within their code. The
judges were additionally impressed by the clarity of the code and supporting
documentation, something which is easily overlooked when writing code. This also
showed in their robot's performance, which was reliably able to navigate around
the central pedestal to collect tokens from the far side.

We also encourage teams to share their progress towards their robots throughout
the year. Team MAI from Gymnasium Markt Indersdorf earned the Online Presence
Award for their ongoing blog posting during both the lead-up to the physical
competition and after lockdown began as well as for their activy on the forums.

For full details on all the awards, please see the [rulebook][rulebook].

You can see a breakdown of scores for each match, as well as the overall league
ranking on the [competition website](/comp/).

Of course, this competition would not be possible without the tireless work of
our [volunteers][volunteers], who developed the [simulated world][simulator] the

> competition was run in and enabled us to switch to a virtual competiton.

Reviewer: I'd be interested in a CTA on this

Author: 5a6a594

Having these tokens within their scoring zone (i.e. their corner of the arena)
at the end of the match would earn them 3 points apiece. However, if a robot
gathered more than one token colour within their scoring zone, each token's
value drops from 3 points to 1 point. Full details of the game and the awards
available this year are available in the [rulebook][rulebook].

Our competition event this year was rather different to normal as COVID-19
prevented our usual weekend event. Instead the competition took place over
three weekends, with matches being live-streamed from within a simulator.
While this changed many aspects of the competition, the core challenge to create
an autonomous robot remained the same.

The Final
---------

This year's final featured four teams who had all proven their capabilities
throughout the competition. All robots got off to a strong start, though
JAMDynamics and Lawrence Sheriff School soon took a lead. However, later in the
match, the "poisoning" effect of having mixed tokens in a scoring zone changed
everything leaving all four teams with the same number of points.

<!-- Video of Final -->
<iframe
  class="center video"

>   src="https://www.youtube-nocookie.com/embed/XXX-TODO"

Reviewer 3: ```suggestion
  src="https://www.youtube-nocookie.com/embed/MAXpgODIqiI"
```

Author: Thanks! Already did this (and removed the other embed) in fcdf8d7.
